STEP_Project/popup.js

```diff
@@ -24,12 +24,13 @@ function findPagecase(givenurl)
   let orgunitspage = givenurl.match(/admin.google.com\/u\/[0-9]+\/ac\/orgunits/g);
   let settingspage = givenurl.match(/admin.google.com\/u\/[0-9]+\/ac\/appsettings\/[0-9]+\/.+/g);
   let onoffpage = givenurl.match(/admin.google.com\/u\/[0-9]+\/ac\/settings\/serviceonoff.*/g);
+  let firstsettingspage = givenurl.match(/admin.google.com\/ac\/appsettings\/[0-9]+\/.+/g);
```

Comment:
This change will be needed for all pages, orgunits, settings and onoff. Suggest you to not create a new variable but modify the existing regexes to optionally incorporate this as you did in pageUrl above.

Reply:
done, regex expressions were modified and extra variable was removed.